{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/utils/models.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import print_function
+
+from django.template.loader import render_to_string
+from django.urls import reverse
+from django.utils.functional import cached_property
+from django.utils.safestring import mark_safe
+from django_lazy_image.templatetags.lazy_image import lazy_image
+from threadlocals.threadlocals import get_current_user
+
+
+class EditableMixin:
+    def get_displayed_text_for_CharField(self, field_name, field_value):
+        # Need to handle max_length
+        return mark_safe(f'<span class="js-SimpleEditable" data-pk="{self.pk}" data-field="{field_name}" data-app="{self._meta.app_label}" data-model="{self._meta.object_name}" contentEditable>{field_value}</span>')
```

Comment:
This would make this entirely unsuitable to be used on user-supplied data; you've marked the entire string as safe which mean that if (for example) an article title contained user-supplied input (see any of our dashboard-type projects) there's a giant XSS here. Use [`format_html`](https://docs.djangoproject.com/en/2.1/ref/utils/#django.utils.html.format_html) for these.